src/test/java/edu/hm/hafner/analysis/IssueTest.java

```diff
@@ -0,0 +1,161 @@
+package edu.hm.hafner.analysis;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests the class {@link Issue}.
+ *
+ * @author Michael Schmid
+ */
+class IssueTest {
+
+    /** Verify an empty issue generated with the builder */
+    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
```

Comment:
JUnitTestMethodWithNoAssertions: In IntelliJ **should** entfernen bei der Inspection